drivers/i2s/i2s_nrfx.c

```diff
@@ -0,0 +1,788 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nordic_nrf_i2s
+
+#include <stdlib.h>
+#include <drivers/i2s.h>
+#include <nrfx_i2s.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(i2s_nrfx, CONFIG_I2S_LOG_LEVEL);
+
+struct queue {
```

Comment:
Can be replaced with `struct k_msgq` instead of custom queue.

Reply:
Applied.

Reply:
@parthitce I addressed your comment. Would you mind taking a look again?

Reply:
@anangl LGTM, thanks